<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScaffoldTab</class>
 <widget class="QWidget" name="ScaffoldTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="layout">
   <item>
    <widget class="QGroupBox" name="scaffold_group">
     <property name="title">
      <string>Project Scaffold</string>
     </property>
     <layout class="QVBoxLayout" name="scaffold_layout">
      <item>
       <widget class="QCheckBox" name="create_scaffold_check">
        <property name="text">
         <string>Create standard project scaffold</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="create_editorconfig_check">
        <property name="text">
         <string>Create .editorconfig file</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="preview_group">
     <property name="title">
      <string>Project Structure Preview</string>
     </property>
     <layout class="QVBoxLayout" name="preview_layout">
      <!-- Custom FolderTreeWidget will be added programmatically -->
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
